client/src/components/Booking/IRequest.tsx

```diff
@@ -0,0 +1,6 @@
+export interface IRequest {
+  status: string;
+  sitterId: string;
+  start: Date;
+  end: Date;
+}
```

Comment:
Let's name this interface more descriptively - `Request` is too generic, maybe use something similar to `BookingRequest` 👍🏼 

Also, there is a folder named `interface` in our project's hierarchy that is dedicated to TS interfaces - we should move this file there. We should also use `.ts` file extension as `.tsx` implies the file contains JSX.